tasks/main.yml

```diff
@@ -58,3 +58,6 @@
 #
 - name: Update facts
   setup:
+
+- name: Verify previous role run results
+  include_tasks: "tests/verify-main.yml"
```

Comment:
We should do this in the test.yml file -- not in the main task file that runs even on customers sites.

Reply:
I'm not sure if variables persist after executing an instance of the role, so we'll have to explicitly add the variables and set the defaults.